RawWaterManagement/RegulationStructure/schema.json

```diff
@@ -5,71 +5,77 @@
     "description": "This entity contains a harmonised description of a generic “regulation structure” made for raw-water supply management domain. Regulation structure consists a junction type object, controlling flow in a channel",
     "type": "object",
     "allOf": [
-              "location": {
-                "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
-              },
-              "position": {
-                  "type": "number",
-                  "measuredFrom":{
-                    "$ref": "../RawWaterManagement-schema.json#/ngsildRelationship"
-                  }
-              },
-              {
-              "properties": {
-                  "type": {
-                      "type": "string",
-                      "enum": ["regulationStructure"],
-                      "description": "NGSI-LD Entity Type"
-                  },
-                  "tag": {
-                    "$ref": "../RawWaterManagement-schema.json#/definitions/tag"
-                  },
-                  "numberOfGates": {
-                    "type": "number",
-                    "description": "Water abstracted (or diverted) from the junction to another source (m3/s)",
-                  },
-                  "numberOfSpillways": {
-                    "type": "number",
-                    "description": "An integer value defining the number of spillways attached to the regulation structure",
-                  },
-                  "telecommand": {
-                    "type": "boolean",
-                    "description": "Define whether the regulation structure is controlled remotely or manually. True/False value",
-                  },
-                  "description": {
-                      "$ref": "../RawWaterManagement-schema.json#/definitions/ngsildProperty"
-                  },
-                  "id": {
-                      "$ref": "../RawWaterManagement-schema.json#/definitions/ngsildProperty"
-                  },
-                  "cross-section": {
-                      "$ref": "../RawWaterManagement-schema.json#/definitions/ngsildProperty"
-                  }
-              }
+      {
+        "position": {
```

Comment:
there is a json schema correct in a previous reply to your comment
this is not valid json schema